gradle/license-report-project.gradle

```diff
@@ -41,12 +41,13 @@ buildscript {
     }
 
     dependencies {
-        classpath deps.build.gradlePlugins.licenseReport
+        classpath 'com.github.jk1:gradle-license-report:1.16'
```

Comment:
So, it's the third place where we have the version hardcoded.

Reply:
It is hard-coded for simplicity and to avoid circular dependencies between `.gradle` and `.gradle.kts` files. 

The `.gradle` part of the code needs to be migrated. Please contribute to this work.